android/app/src/main/java/com/microsoft/codepush/react/CodePushTelemetryManager.java

```diff
@@ -113,13 +113,15 @@ public WritableMap getUpdateReport(WritableMap currentPackage) {
     }
 
     public void recordStatusReported(ReadableMap statusReport) {
-        if (statusReport.hasKey(STATUS_KEY) && DEPLOYMENT_SUCCEEDED_STATUS.equals(statusReport.getString(STATUS_KEY))) {
-            if (statusReport.hasKey(APP_VERSION_KEY)) {
-                saveStatusReportedForIdentifier(statusReport.getString(APP_VERSION_KEY));
-            } else if (statusReport.hasKey(PACKAGE_KEY)) {
-                String packageIdentifier = getPackageStatusReportIdentifier(statusReport.getMap(PACKAGE_KEY));
-                saveStatusReportedForIdentifier(packageIdentifier);
-            }
+        // We don't need to record rollback reports, so exit early if that's what was specified.
+        if (statusReport.hasKey(STATUS_KEY) && DEPLOYMENT_FAILED_STATUS.equals(statusReport.getString(STATUS_KEY)))
```

Comment:
I'm not sure if the original bug exists in Cordova. If so, then yeah, we should add it there as well.